client/my-sites/site-settings/redirect-to/README.md

```diff
@@ -0,0 +1,49 @@
+# Redirect To
+
+This component provides automatic (prior to rendering the site) redirection for sites with no access to a particular page.
+Here, we consider Atomic sites to be Jetpack-like. Note: although Atomic sites are also Jetpack sites, they need to be treated as non-Jetpack sites in certain situations, example of which is access to the (dis)connection actions.
+
+## Props passed to the HoC
+### `WrappedComponent`
+
+Component, which we want to render using this HoC. When not apt for rendering
+due to lack of access rights (non-Jetpack or Atomic), the redirection will be handled using default or provided route (see below), e.g. the `ManageConnection` component in Site Settings.
+
+
+### `redirectRoute` ( *optional* )
+
+Destination site route passed to the HoC. The default redirect fallback is `/settings/general/:siteSlug`, where `:siteSlug` is the user site.
+
+### How to use:
+
+```js
+import redirectNonJetpack from 'my-sites/site-settings/redirect-to';
+
+export default flowRight(
+	localize,
+	redirectNonJetpack( redirectRoute )
```

Comment:
Maybe for the sake of the example you can define a `redirectRoute` or remove it if you want to demonstrate how the default value works?

Reply:
yes!